spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -120,6 +121,26 @@
         'PresentationChanged' => 'false',
       )
     end
+
+    it 'notes that address line 2 was present' do
+      expect(response.to_h).to include(address_line2_present: true)
+    end
+  end
+
+  context 'when there is no address line 2' do
+    body_no_line2 = JSON.parse(LexisNexisFixtures.true_id_response_success_2).tap do |json|
+      json['Products'].first['ParameterDetails'] = json['Products'].first['ParameterDetails'].
+        select { |f| f['Name'] != 'Fields_AddressLine2' }
+    end.to_json
```

Comment:
usually we don't define variables at context-scope? I would either put this inside its own `let` or move it inside the block for `let(:success_response_noline2)